charts/kong/example-values/minimal-k4k8s-enterprise.yaml

```diff
@@ -0,0 +1,21 @@
+# Basic values.yaml for Kong for Kubernetes Enterprise
+# Several settings (search for the string "you-must") require user-provided
+# Secrets. These Secrets must be created before installation.
+
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 2.0.4.1-centos
+  pullSecrets:
+    # you-must add this: https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-docker-registry-access
+    - kong-enterprise-k8s-docker
+
+enterprise:
+  enabled: true
+  license_secret: you-must-create-a-license-secret
```

Comment:
When I meant that replace the secret names by the names we use in our guides, I meant it for all secrets. Please change this to how you changed the image pull secret.